extensions/positron-python/package.json

```diff
@@ -541,7 +541,7 @@
                         }
                     ],
                     "markdownDescription": "%python.environmentProviders.enable.markdownDescription%",
-                    "scope": "machine",
+                    "scope": "machine-overridable",
```

Comment:
I've changed this to machine-overridable so the user can enable/disable Venv and Conda at the workspace level as well. It's not path-dependent like the other settings.